src/api/spec/components/previews/reports_notice_component_preview.rb

```diff
@@ -0,0 +1,20 @@
+class ReportsNoticeComponentPreview < ViewComponent::Preview
+  # Preview at http://HOST:PORT/rails/view_components/reports_notice_component/user_reportable_as_admin
+  def user_reportable_as_admin
+    reportable = User.last
+    User.take(3).each do |reporter|
+      Report.new(reportable:, user: reporter).save!
+    end
```

Comment:
I think this could work:

```suggestion
    User.take(3).each do |user|
      Report.new(reportable:, user:).save!
    end
```

... but also to change the name of the parameter from `user` to `reporter`:

```suggestion
    User.take(3).each do |reporter|
      Report.new(reportable:, reporter:).save!
    end
```

Now we have kind of a mix of both. What do you think?